src/website/views/custom-elements/alert.pug

```diff
@@ -476,36 +334,23 @@ p.-text
 .example.-mb--3
   .-p--1
     chi-alert.-m--2(type='banner', icon='warning', color='warning', dismissible)
-      .m-alert__content
-        p.m-alert__text This is a dismissible warning alert
-    chi-alert.-m--2(type='banner', color='warning', icon='warning', dismissible)
-      .m-alert__content
-        p.m-alert__title Warning
-        p.m-alert__text This is a dismissible warning alert
-    chi-alert.-m--2(type='banner', color='warning', icon='warning', size='lg', dismissible)
-      .m-alert__content
-        p.m-alert__title Warning
-        p.m-alert__text This is a large dismissible warning alert
+      p.m-alert__text This is a dismissible warning alert
+    chi-alert.-m--2(type='banner', color='warning', icon='warning', alert-title="Warning", dismissible)
+      p.m-alert__text This is a dismissible warning alert
+    chi-alert.-m--2(type='banner', color='warning', icon='warning', size='lg', alert-title="Warning", dismissible)
+      p.m-alert__text This is a large dismissible warning alert
   :code(lang="html")
     <!-- Dismissible -->
     <chi-alert type="banner" color="warning" icon="warning" dismissible>
-      <div class="m-alert__content">
-        <p class="m-alert__text">This is a dismissible warning alert</p>
-      </div>
+      <p class="m-alert__text">This is a dismissible warning alert</p>
     </chi-alert>
     <!-- Dismissible Titled -->
-    <chi-alert type="banner" color="warning" icon="warning" dismissible>
-      <div class="m-alert__content">
-        <p class="m-alert__title">Warning</p>
-        <p class="m-alert__text">This is a dismissible warning alert</p>
-      </div>
+    <chi-alert type="banner" color="warning" icon="warning" alert-title="Warning" dismissible>
+      <p class="m-alert__text">This is a dismissible warning alert</p>
     </chi-alert>
     <!-- Large Dismissible Titled -->
-    <chi-alert type="banner" color="warning" icon="warning" size="lg" dismissible>
-      <div class="m-alert__content">
-        <p class="m-alert__title">Warning</p>
-        <p class="m-alert__text">This is a large dismissible warning alert</p>
-      </div>
+    <chi-alert type="banner" color="warning" icon="warning" size="lg" alert-title="Warning" dismissible>
+      <p class="m-alert__text">This is a large dismissible warning alert</p>
     </chi-alert>
 
 h4 Actionable
```

Comment:
@janibolkvadze Line 370. The Banner Actionable example code snippet is using old format. Can you fix and I will merge?

Reply:
On it